src/Bicep.Cli/Commands/TestCommand.cs

```diff
@@ -75,5 +74,32 @@ public async Task<int> RunAsync(TestArguments args)
         }
 
         private bool IsBicepFile(string inputPath) => PathHelper.HasBicepExtension(PathHelper.FilePathToFileUrl(inputPath));
+        private void LogResults(Validation validation){
+            foreach(var (name, evaluation) in validation.SuccessfullEvaluations){
+                logger.LogInformation($"[✓] Evaluation {name} Passed!");
+            }
+            foreach(var (name, evaluation) in validation.SkippedEvaluations){
+                logger.LogError($"[-] Evaluation {name} Skipped!");
+                logger.LogError($"Reason: {evaluation.Error?.Message}");
+
+            }
+            foreach(var (name, evaluation) in validation.FailedEvaluations){
+                logger.LogError($"[✗] Evaluation {name} Failed at {evaluation.FailedAssertions.Count} / {evaluation.Assertions.Count} assertions!");
+                foreach(var (assertion, _) in evaluation.FailedAssertions){
+                    logger.LogError($"\t[✗] Assertion {assertion} failed!");
```

Comment:
It might be worth making the symbols indicating the test results (`[✓]`, `[✗]`, and `[-]`) constant strings.